fr/core-libraries/components/flash.rst

```diff
@@ -0,0 +1,78 @@
+Flash
+#####
+
+.. php:class:: FlashComponent(ComponentCollection $collection, array $config = [])
+
+FlashComponent provides a way to set one-time notification messages to be
+displayed after processing a form or acknowledging data. CakePHP refers to these
+messages as "flash messages". FlashComponent writes flash messages to
+``$_SESSION``, to be rendered in a View using
+:doc:`FlashHelper </core-libraries/helpers/flash>`.
+
```

Comment:
A short note that this component and its sister helper succeed the older ones would make it more clear for people having some knowledge about the old ones but haven't read the migration guide.